app/models/order.rb

```diff
@@ -26,6 +26,11 @@ def send_order_to_bakery
 
   def self.completed_orders
     response = HTTParty.get("https://cakewalkers-api.herokuapp.com/bake_jobs")
-    @orders_for_delivery = response.select { |order| order["state"] == "done" }
+    orders_for_delivery = response.select { |order| order["state"] == "done" }
+    orders_for_delivery.each do |order|
```

Comment:
Is this tested? orders_for_delivery are JSON objects, they will not respond to `Order#ready_at`